### PR TITLE
Update Localizable.strings

### PR DESCRIPTION
**Pull request by marinotws**
new option in Settings -> Automatic upload of camera images

### CODE REVIEW

**Comment by marinotws** [comment]
@MorrisJobke commit please.

File: en.lproj/Localizable.strings [hunk 1]
```diff
@@ -140,6 +140,8 @@
 "_camera_upload_not_select_home_"   = "Select a folder";
 "_upload_camera_save_album_"        = "Copy photo or video in the photo album";
 "_upload_camera_save_album_footer_" = "After the automatic sending of the photo or video, a copy is saved in the photo album : _brand_ (does not require more space in the camera roll)";
+"_upload_camera_fullphotos_"        = "Upload all camera photos/videos";
+"_upload_camera_fullphotos_footer_" = "Select above the options before the start";
```
Comment (MorrisJobke) [inline en.lproj/Localizable.strings]: "Select the options above"

Before starting what? Start the upload?

Comment (MorrisJobke) [inline en.lproj/Localizable.strings]: If so then:

"Select the options above before uploading"

**Comment by marinotws** [comment]
yes

File: en.lproj/Localizable.strings (hunk 1 above)
Comment (marinotws) [inline en.lproj/Localizable.strings]: perfect

**Pull request merged**